### PR TITLE
Selector Issues

### PR DESCRIPTION
**Pull request by pfistfl**
Solves the following issues:
- Autoconvert selector `po(selector_xxx())`: https://github.com/mlr-org/mlr3pipelines/issues/402
- Use selector for `$.select_cols` https://github.com/mlr-org/mlr3pipelines/issues/320
- `cardinality selector`Selector for factorials by cardinality https://github.com/mlr-org/mlr3pipelines/issues/330
- `missind` now selects everything except for `factor, character, ordinal` #395

### CODE REVIEW

File: R/PipeOpImputeHist.R [hunk 1]
```diff
@@ -60,7 +60,7 @@ PipeOpImputeHist = R6Class("PipeOpImputeHist",
   ),
   private = list(
 
-    .select_cols = function(task) task$feature_types[get("type") %in% c("numeric", "integer"), get("id")],
+    .select_cols = function(task) selector_type(c("numeric", "integer"))(task),
```
Comment (mb706) [inline R/PipeOpImputeHist.R:63]: I *think* you could jsut do `.select_cols = selector_type(c("numeric", "integer"))` here, but I'm not 100% sure. Have you tried that and it failed for some reason?

Comment (pfistfl) [reply R/PipeOpImputeHist.R:63]: So I have not fully understood why, but it fails.

What happens is, that the `types` supplied to `slector_type` are lost after construction.
and I get `Object 'types' not found.`

Comment (mb706) [reply R/PipeOpImputeHist.R:63]: interesting, probably R6 takes away environments of methods

Comment (mb706) [reply R/PipeOpImputeHist.R:63]: https://github.com/r-lib/R6/issues/80

**Pull request merged**